index.html

```diff
@@ -31,13 +32,210 @@ <h2 id="job-title">I am a Software Developer</h2>
+                    <div class="floating">
+                        <div>
+                            <h3 id="project-title">Profesional Art Printing Data</h3>
+                        </div>
+                        <div>
+                            <p id="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard</p>
+                        </div>
+                        <div>
+                            <a href="#"><button class="project-1-buttons">html</button></a>
+                            <a href="#"><button class="project-1-buttons" >bootstrap</button></a>
+                            <a href="#"><button class="project-1-buttons">Ruby</button></a>
+                        </div>
+                    </div>
+                    <div class="see-project-div">
+                        <a href=""><button id="see-project-button">See Project</button></a>
+                    </div>
+                </div>
+                
+            </section>
+            <section class="about-me">
+                <div id="about-div">
+                    <h2 id="about-me-title">About Me</h2>
+                </div>
+                <div id="about-me-description-div">
+                    <p id="about-me-description">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                    <a href=""><button id="get-resume-button">Get My Resume</button></a>
+                </div>
+                <p id="line3"></p>
+                <div class="languages-div">
+                    <div class="languages">
+                        <div class="ul-image"><img id="frameworks-image" img src="images/Rectangle 55 (1).png" alt=""><p id="framework-text">Language </p></div>
+                            <li id="individual-languages">JavaSript</li>
+                            <li id="individual-languages">Ruby</li>
+                            <li id="individual-languages">Html</li>
+                            <li id="individual-languages">Css</li>
+                    </div>
+                    <div class="frameworks-div">
+                        <div class="ul-image"><img id="frameworks-image" img src="images/Rectangle 56 (1).png" alt=""><p id="framework-text">Frameworks </p></div>
+                            <div class="mini-language-div">
+                                <li id="individual-frameworks">Bootstrap</li>
+                                <li id="individual-frameworks">Ruby on Rails</li>
+                                <li id="individual-frameworks">RSPec</li>
+                                <li  id="individual-frameworks">CapyBara</li>
+                                <li id="individual-frameworks">Selenium</li>
+                            </div>
```

Comment:
Kindly, when you correct you deleted the UL tag, check the following documentation and apply it to all your UL listings on your page

https://www.w3schools.com/html/html_lists.asp